catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/OperationsMetacardSupport.java

```diff
@@ -140,6 +140,7 @@ void generateMetacardAndContentItems(List<ContentItem> incomingContentItems,
                 metacardMap.put(metacard.getId(), metacard);
 
                 ContentItem generatedContentItem = new ContentItemImpl(metacard.getId(),
+                        StringUtils.isNotEmpty(contentItem.getQualifier()) ? contentItem.getQualifier() : "",
```

Comment:
This was required to correctly identify incoming updates that already had qualifiers. For instance, when we submitted qualified content back to the Catalog after asynchronous processing finished, the content was not receiving the correct qualifier and would not correctly hookup in the Metacard.